pkg/kubernetes/scripts/projects.js

```diff
@@ -940,10 +977,13 @@
 
                 return methods.check(request, { "metadata.name": "#project-new-name" })
                     .then(function() {
-                        return methods.create(request)
-                            .then(function() {
-                                return projectData.shareImages(name, fields.access === "shared");
-                            });
+                        return methods.create(request);
+                    })
+                    .then(function(){
+                        return projectData.shareImages(name, fields.access === "shared");
+                    })
+                    .then(function(){
+                        return projectData.makeRegistryAdmin(name, dialogData.currUser);
```

Comment:
> IMO we need to add a project template which will not create a 'admin' policy. ok ?

Once this is merged and we know more about how this will be deployed we will adjust as needed. With this work we have less need to have special project templates. This is a really positive direction.